src/main/java/com/mowmaster/rprocessing/tiles/TileClayBloomery.java

```diff
@@ -23,9 +27,8 @@
     public int carboncount=0;
     public int maxcarbon=8;
     public int maxore=8;
-    public int orecount=0;
-    public int oreiron=0;
-    public int oregold=0;
+    public final int maxore = 8;
```

Comment:
It's a constant, it probably shouldn't be changing. Really in this case it's just code style, though.